### PR TITLE
Add support to install from aka.ms/lis

### LINKED ISSUE #365: CustomLIS: Add support to install from aka.ms/lis
Current -CustomLIS option only supports -
1. lisnext
2. tar.gz downloadable url.

We need to add support for downloading from aka.ms/lis link and install it.

### PR DESCRIPTION
**Pull request by sireeshapotnuru**
Fixes #365 

This PR is to add support to install lis from aka.ms/lis using -CustomLIS
Usage ; -CustomLIS "LatestLIS"

### CODE REVIEW

**Comment by sireeshapotnuru** [comment]
07/08/2019 11:49:22 : [INFO ]
[LISAv2 Test Results Summary]
Test Run On           : 07/08/2019 11:41:49
VHD Under Test        : \\redmond\wsscfs\OSTC\LIS\VHD\Cloudbase\RHEL\RHEL_7.3\rhel73_x64.vhdx
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:7

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes)
-------------------------------------------------------------------------------------------------------------------------------------------
    1 LIS_DEPLOY           LIS-DEPLOY-SCENARIO-1                                                             PASS                 5.44
        Install-LIS-Scenario-1 : PASS

[LISAv2 Test Results Summary]
Test Run On           : 07/08/2019 12:08:25
VHD Under Test        : \\redmond\wsscfs\OSTC\LIS\VHD\Cloudbase\RHEL\RHEL_7.3\rhel73_x64.vhdx
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:7

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes)
-------------------------------------------------------------------------------------------------------------------------------------------
    1 CORE                 VERIFY-DEPLOYMENT-PROVISION                                                       PASS                 2.44
        FirstBoot : PASS
        FirstBoot : Call Trace Verification : PASS
        Reboot : PASS
        Reboot : Call Trace Verification : PASS

File: Libraries/CommonFunctions.psm1 [hunk 1]
```diff
@@ -647,9 +647,9 @@ function Install-CustomLIS ($CustomLIS, $customLISBranch, $allVMData, [switch]$R
 	try
 	{
 		$CustomLIS = $CustomLIS.Trim()
-		if ( ($CustomLIS -ne "lisnext") -and !($CustomLIS.EndsWith("tar.gz")))
+		if ( ($CustomLIS -ne "lisnext") -and !($CustomLIS.EndsWith("tar.gz")) -and ($CustomLIS -ne "latest"))
```
Comment (johnsongeorge-w) [inline Libraries/CommonFunctions.psm1]: Use meaningful option. Use LatestLIS instead of Latest.

**Review by johnsongeorge-w — CHANGES_REQUESTED** [review]

File: Libraries/CommonFunctions.psm1 (hunk 1 above)
Comment (sireeshapotnuru) [reply Libraries/CommonFunctions.psm1]: changed to LatestLIS

File: Testscripts/Windows/LIS-INSTALL-SCENARIOS.ps1 [hunk 2]
```diff
@@ -489,6 +489,12 @@ Function Main {
                 $LISTarballUrlOld = $param.Replace("LIS_TARBALL_URL_OLD=","")
             }
         }
+        if( $LISTarballUrlCurrent -eq "LatestLIS" ) {
+            $LISTarballUrlCurrent=Run-LinuxCmd -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -username $user -password $password -command "echo ``curl -Ls -o /dev/null -w %{url_effective} http://aka.ms/lis``"
+        }
+        if( $LISTarballUrlOld -eq "LatestLIS" ) {
```
Comment (chvalean) [inline Testscripts/Windows/LIS-INSTALL-SCENARIOS.ps1:495]: how can the old LIS be the latest version?

Comment (sireeshapotnuru) [reply Testscripts/Windows/LIS-INSTALL-SCENARIOS.ps1:495]: When Creating a new LIS RPM build,
LatestLIS will be LISTarballUrlOld and the generated LIS will be LISTarballUrlCurrent

**Review by chvalean — APPROVED** [review]

**Review by johnsongeorge-w — APPROVED** [review]

**Review by iamshital — APPROVED** [review]

**Pull request merged**